pyteal/ir/ops.py

```diff
@@ -170,6 +170,10 @@ def min_version(self) -> int:
     gtxnas              = OpType("gtxnas",              Mode.Signature | Mode.Application, 5)
     gtxnsas             = OpType("gtxnsas",             Mode.Signature | Mode.Application, 5)
     args                = OpType("args",                Mode.Signature,                    5)
+    itxn_next           = OpType("itxn_next",           Mode.Application,                  6)
```

Comment:
nit: move `itxn_next` into the `itxn_*` section above. Admittedly this puts a v6 op ahead of a v5 op. Is that a problem?

Reply:
never mind... after thinking more about this, I convinced myself of the opposite...

Reply:
👍